src/uct/ib/rdmacm/rdmacm_cm.c

```diff
@@ -164,10 +165,9 @@ static ucs_status_t uct_rdmacm_cm_id_to_dev_addr(struct rdma_cm_id *cm_id,
         return UCS_ERR_NO_MEMORY;
     }
 
-    uct_ib_address_pack(&qp_attr.ah_attr.grh.dgid, qp_attr.ah_attr.dlid,
-                        IBV_PORT_IS_LINK_LAYER_ETHERNET(&port_attr),
-                        qp_attr.ah_attr.is_global,
-                        dev_addr);
+    uct_ib_address_pack(qp_attr.ah_attr.is_global ? &qp_attr.ah_attr.grh.dgid : NULL,
```

Comment:
seems redundant, why not pass is_global instead of 0 as "is_global_addr"?

Reply:
because ```is_global_addr``` is force GRH for iface call and "memory is accessible" for CM - different interpretation of argument and should be different behavior

Reply:
I just unified behavior